pbsmetrics/prometheus/prometheus.go

```diff
@@ -58,6 +59,9 @@ func NewMetrics(cfg config.PrometheusMetrics) *Metrics {
 	timerBuckets := prometheus.LinearBuckets(0.05, 0.05, 20)
 	timerBuckets = append(timerBuckets, []float64{1.5, 2.0, 3.0, 5.0, 10.0, 50.0}...)
 
+	timerBucketsQuickTasks := prometheus.LinearBuckets(0.005, 0.005, 20)
```

Comment:
Prebid Cache times may be very fast. I defined a new time bucket for measuring tasks which are expected to be quick. The series looks like: 1ms, 1.5ms, 3ms, 5ms, 10ms, 15ms, 20ms, 25ms, 30ms, 35ms, 40ms, 45ms, 50ms, 55ms, 60ms, 65ms, 70ms, 75ms, 80ms, 85ms, 90ms, 95ms, 100ms.